autosynth/multi.py

```diff
@@ -129,12 +129,16 @@ def _close_issue(gh, result, existing_issue):
     )
 
 
+def _last_n_result_characters(proc: subprocess.CompletedProcess, n: int) -> str:
+    return proc.stdout.decode("utf-8").strip()[-n:]
+
+
 def _file_or_comment_on_issue(gh, result, issue_title, existing_issue):
     message = f"""\
 Here's the output from running `synth.py`:
 
 ```
-{result['result'].stdout}
+{_last_n_result_characters(result['result'], 1000)}
```

Comment:
The previous code didn't call `.decode("utf-8")` so I'm not really sure of the consequences of decoding the output.
To me, `[-n:]` is actually clearer than calling a function called `_last_n_result_characters`.
The error message told us the limit was 64k, so let's be a little more generous.

Altogether, that yields a comment and a line of code:
```
# Github rejects issues with bodies > 64k.
{result['result'].stdout[-10000:]}
```

I suppose there's the chance we'll slice the string on a non-utf8 character boundary, but that's very unlikely because we're slicing an ascii-encoded text log, and it would be inconsequential even if it happened.

Reply:
I was attempting to test the changes and be more confident in the types that were being used. I can switch to this simpler version.